pkg/services/authz/zanzana/common/info.go

```diff
@@ -133,7 +139,7 @@ func (r ResourceInfo) Type() string {
 }
 
 func (r ResourceInfo) Context() *structpb.Struct {
-	if !r.IsGeneric() {
+	if !r.IsGeneric() && r.typ != TypeFolder && r.typ != TypeTeam {
```

Comment:
```suggestion
```

As we talked about we don't need this check, not sure why I added it in the first place